### DISCUSSION TITLE
Does anyone feel that OpenSSL 3.4 is slower than OpenSSL 3.0?

### DISCUSSION THREAD

**Discussion by freebendy** [openssl/openssl#26259] [General Discussion]
I update OpenSSL from 3.0.14 to 3.4.0 on my i.mx6ul embedded device. I can feel that the OpenSSL command line is much slower than 3.0.14 even just a simple "openssl version" command.

**Comment by nhorman** [comment]
do you have any data to support that?

**Comment by mattcaswell** [comment]
FYI, you can look at our performance dashboard here:

https://openssl-library.org/performance.html

That shows 3.4 outperforming (or at least equalling) 3.0 in every test. I definitely would not expect a noticeable slow down just running `openssl version`. This most likely suggests some local problem.
